src/pages/lessons/[slug].tsx

```diff
@@ -560,7 +580,11 @@ const Lesson: React.FC<LessonProps> = ({initialLesson}) => {
                   <GoProCtaOverlay
                     lesson={lesson}
                     viewLesson={() => {
-                      router.reload()
+                      send({
+                        type: 'LOAD',
+                        lesson: initialLesson,
+                        viewer,
+                      })
```

Comment:
Get the user into the `viewing` state with a transition instead of a page reload.